pkg/resourcemanager/webhook/highavailabilityconfig/handler.go

```diff
@@ -367,3 +446,46 @@ func (h *Handler) mutateTopologySpreadConstraints(
 		podTemplateSpec.Spec.TopologySpreadConstraints = append(filteredConstraints, constraints...)
 	}
 }
+
+func mutateAutoscalingReplicas(
+	log logr.Logger,
+	failureToleranceType *gardencorev1beta1.FailureToleranceType,
+	obj client.Object,
+	getMinReplicas func() *int32,
+	mutateMinReplicas func(*int32),
```

Comment:
```suggestion
	setMinReplicas func(*int32),
```
To keep it consistent with the change in the `mutateReplicas()` function